pkg/util/testutil/docker/run.go

```diff
@@ -44,8 +44,8 @@ func Run(t testing.TB, cfg LifecycleConfig) error {
 			t.Logf("%s command succeeded. %s container is running", cfg.command(), cfg.Name())
 			return nil
 		}
+		t.Logf("[Attempt #%v] failed to start %s server: %v", i, cfg.Name(), err)
```

Comment:
i runs from 0 to N-1
```suggestion
		t.Logf("[Attempt #%v] failed to start %s server: %v", i+1, cfg.Name(), err)
```